package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0-beta1",
   "description": "Node.js GitHub Bot",
   "scripts": {
-    "start": "node server.js",
+    "start": "node server.js | bunyan -o short",
```

Comment:
will this work as-is? `bunyan` wouldn't necessarily live in path; perhaps `node_modules/.bin/bunyan` ?